raiden/raiden_service.py

```diff
@@ -634,6 +617,26 @@ def settle(self, token_address, partner_address):
         netting_channel.settle()
         return channel
 
+    def get_channel_new_events(self, token_address, from_block, to_block=''):
```

Comment:
Because I consider it that you always need to specify a starting block for events but the default for the `to_block` is the latest.

I suppose this should also be documented in the `api.rst`.